Database/Repositories/Interfaces/ITournamentsRepository.cs

```diff
@@ -54,18 +55,21 @@ public interface ITournamentsRepository : IRepository<Tournament>
     /// <param name="pageSize">The size of the collection</param>
     /// <param name="querySortType">Determines how the results are sorted</param>
     /// <param name="descending">Whether to sort the results in descending order</param>
-    /// <param name="verified">
-    /// Whether the resulting tournaments must be verified and have completed processing
-    /// </param>
-    /// <param name="ruleset">An optional ruleset to filter by</param>
-    /// <remarks>All returned entities will not be tracked by the context</remarks>
-    Task<ICollection<Tournament>> GetAsync(
-        int page,
+    /// <remarks>None of the entities returned will be tracked by the context</remarks>
+    Task<ICollection<Tournament>> GetAsync(int page,
```

Comment:
Missing docs for the new parameters

Reply:
Docs would simply be copy / pasted from the DTO and then have to be updated in parallel. Honestly I really really don't like doing this in the first place but it's more of my way of avoiding use of the ugly `Api{Entity}Repository` we have going on right now (since we cannot pass the DTO directly).

Reply:
I agree but should still be documented.